src/gc.h

```diff
@@ -257,6 +258,18 @@ typedef struct {
     pagetable1_t *meta1[REGION2_PG_COUNT];
 } pagetable_t;
 
+typedef struct {
+    _Atomic(size_t) bytes_mapped;
```

Comment:
So this metric might not be super useful, I was checking mapped vs decommited memory, but we revive the madvised pages, so I'm super sure how to make this metric accurate. One idea is to just track max amount of memory mapped because since we revive pages, we only go to `mmap` to actually increase heap.

Reply:
It made more sense on the old allocator.